api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -769,88 +768,6 @@ public void test_maybeSendAccessExpirationEmail_extra_expired() {
     verifyZeroInteractions(mailService);
   }
 
-  @Test
-  public void test_getRegisteredTierExpirations_empty() {
-    assertThat(userService.getRegisteredTierExpirations()).isEmpty();
-  }
-
-  @Test
-  public void test_getRegisteredTierExpirations_one_year() {
```

Comment:
Unsure about these test cases - i.e. whether they were intentionally targeting the method I removed, or if they were covering some deeper behavior which we should maintain coverage of